connector/routingconnector/config.go

```diff
@@ -39,6 +39,10 @@ type Config struct {
 	// Table contains the routing table for this processor.
 	// Required.
 	Table []RoutingTableItem `mapstructure:"table"`
+
+	// MatchOnce determines whether the connector matches multiple statements.
+	// Optional.
+	MatchOnce bool `mapstructure:"match_once"`
```

Comment:
Can we also set `MatchOnce` to `false` by default in the `factory.go:createDefaultConfig()` method? I think it would be best to be explicit.

Reply:
I disagree here. Booleans have false as their zero value. We typically do not explicitly declare zero values for common data types.

Reply:
That's fair, especially if existing common practice here is to not explicitly set it.